packages/conf-libelf/conf-libelf.1.0.0/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+authors: ["Bayge <alex@fluidity.money>"]
+maintainer: "https://github.com/ocaml/opam-repository/issues"
+license: "LGPL-2.0-or-later"
+homepage: "https://github.com/ocaml/opam-repository"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+build: ["pkg-config" "--exists" "libelf"]
+depexts: [
+  ["libelf-dev"] {os-family = "debian" | os-family = "ubuntu"}
+  ["elfutils-libelf-devel"] {os-distribution = "fedora" | os-family = "fedora"}
+  ["libelf-devel"] {os-family = "suse" | os-family = "opensuse"}
+  ["libelf"] {os-distribution = "arch"}
+  ["dev-libs/libelf"] {os-distribution = "gentoo"}
+  ["libelf"] {os = "freebsd"}
+  ["libelf"] {os = "macos" & os-distribution = "homebrew"}
+  ["libelf-devel"] {os = "win32" & os-distribution = "cygwinports"}
+  ["elfutils-dev"] {os = "alpine"}
```

Comment:
```suggestion
  ["elfutils-dev"] {os-distribution = "alpine"}
```

Argh, Alpine is still triggering red CI lights unfortunately. The fix is simple though.
Sorry I missed this the last time around! :shrug: :grimacing:

Reply:
@jmid, merged now! Thank you for all the legwork on this.